etc/nebula-metad.conf.default

```diff
@@ -15,6 +15,8 @@
 --logbufsecs=0
 
 ########## networking ##########
+# List of meta server addresses, the format looks like ip1:port1, ip2:port2, ip3:port3
```

Comment:
It's good to add the 'meta_server_addrs' configure item to the nebula-metad.conf.